### ISSUE TITLE
AJAX requests / move towards no reloads on game page.

### ISSUE THREAD

**Issue by aletson** [aletson/votefinder-web#80]
Some things...like adding player, replacing players, etc., should take place without requiring a page reload. This will require JS in place to handle this but shouldn't be difficult. Still should wait till post 3.0 because this is probably gonna need reworks after #50 anyway

**Comment by velinath** [comment]
#61 manual votes too

**Comment by aletson** [comment]
#128 , also wontfix
